src/routers/workspaces.py

```
@@ -53,10 +53,10 @@ async def get_or_create_workspace(
     dependencies=[Depends(require_auth(admin=True))],
 )
 async def get_all_workspaces(
-    options: Optional[schemas.WorkspaceGet] = Body(
+    options: schemas.WorkspaceGet | None = Body(
         None, description="Filtering and pagination options for the workspaces list"
     ),
```

Ruff (0.11.9)

Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)